1.8/usage/haproxy.md

```diff
@@ -0,0 +1,77 @@
+---
+post_title: Configuring HTTP Proxy in Front of DC/OS
+nav_title: Configure HTTP Proxy
+menu_order: 61
+---
+
+You can configure a secure communication channel between HAProxy and the DC/OS Admin Router, using a custom hostname and port.
```

Comment:
In your changeset, you have introduced this sentence:

> You can configure a secure communication channel from a custom hostname and port to DC/OS Admin Router by using an HTTP Proxy.

"from a custom hostname and port to DC/OS Admin Router" does not make sense, technically. 

Can we, like I proposed, write something like "Sometimes it is required to put an HTTP proxy in front of DC/OS Admin Router." to motivate this docs page? The point is that we do _not know_ why customers want to place an HTTP proxy in front of Admin Router. They may have various good (or bad :-)) reasons. The point of this docs page really is: _if_ they need to do so (for whatever reason), then we can help them.